glean-core/src/core_metrics.rs

```diff
@@ -26,6 +26,8 @@ pub struct ClientInfoMetrics {
     pub channel: Option<String>,
     /// The Android specific SDK version of the software running on this hardware device (e.g. "23").
     pub android_sdk_version: Option<String>,
+    /// The Windows specific OS build version (e.g. "19043")
+    pub windows_build_number: Option<String>,
```

Comment:
```suggestion
    pub windows_build_number: Option<i32>,
```

double-check how we send that on legacy telemetry.